packages/@vue/cli-ui/README.md

```diff
@@ -1,15 +1,43 @@
 # @vue/cli-ui
 
-Start development version:
+### Local development
+
+Before starting a local cli-ui instance, 
+consider following the [contributing guide](https://github.com/vuejs/vue-cli/blob/dev/.github/CONTRIBUTING.md) 
+in order to download all required dependencies of vue-cli's packages.
+ 
+Just after, you should build `@vue/cli-ui-addon-webpack` by running:
+```bash
+cd ../cli-ui-addon-webpack
+yarn build
```

Comment:
Instead: run `yarn serve` in a 3rd terminal.

Reply:
Hum, I think I will revert to `yarn build`. Using `yarn serve` give me this error during tests:

```
Error: Cannot find module '@vue/cli-ui-addon-webpack/dist'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:548:15)
    at Function.resolve (internal/modules/cjs/helpers.js:18:19)
    at serve (/home/kocal/Dev/vue-cli/packages/@vue/cli-ui/apollo-server/connectors/client-addons.js:56:34)
    at Layer.handle [as handle_request] (/home/kocal/Dev/vue-cli/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/kocal/Dev/vue-cli/node_modules/express/lib/router/index.js:317:13)
    at /home/kocal/Dev/vue-cli/node_modules/express/lib/router/index.js:284:7
    at param (/home/kocal/Dev/vue-cli/node_modules/express/lib/router/index.js:354:14)
    at param (/home/kocal/Dev/vue-cli/node_modules/express/lib/router/index.js:365:14)
    at param (/home/kocal/Dev/vue-cli/node_modules/express/lib/router/index.js:365:14)
    at Function.process_params (/home/kocal/Dev/vue-cli/node_modules/express/lib/router/index.js:410:3)
```

Reply:
For the tests you need to build the webpack addon at least once.

Reply:
So what's the point for using `yarn serve`, since it doesn't output anything in `dist` folder?

Reply:
It's serving the hot-reloaded webpack dashboard components on localhost which are then loaded in the UI when run in dev mode.

Reply:
Updated.